init-tools.sh

```diff
@@ -25,6 +25,12 @@ OSName=$(uname -s)
             OS=OSX
             __DOTNET_PKG=dotnet-dev-osx-x64
             ulimit -n 2048
+	    # Format x.y.z as single integer with three digits for each part
```

Comment:
nit: Align this with the above block, it looks like it's not indented far enough.